samples/image-metadata/src/components/AppView/AppView.js

```diff
@@ -0,0 +1,156 @@
+      this.props.sdk.notifier.error(
+        `Id "${contentTypeId}" is taken. Try a different name for the content type`
+      );
+      return false;
+    }
+
+    const { sdk } = this.props;
+    const contentType = await sdk.space
+      .createContentType({
+        sys: {
+          id: contentTypeId
+        },
+        name: contentTypeName,
+        fields: [
+          {
+            id: 'title',
+            name: 'Title',
+            required: true,
+            type: 'Symbol'
+          },
+          {
+            id: 'image',
+            name: 'Image',
+            required: true,
+            type: 'Link',
+            linkType: 'Asset'
+          },
+          {
+            id: 'imageMetadata',
+            name: 'Image metadata',
+            required: true,
+            type: 'Object'
+          }
+        ]
+      })
+      .catch(() =>
+        this.props.sdk.notifier.error(`Failed to create content type "${contentTypeName}"`)
+      );
+
+    // Set the newly created content type's state to "Published"
+    await sdk.space
+      .updateContentType(contentType)
+      .catch(() =>
+        this.props.sdk.notifier.error(`Failed to publish content type "${contentTypeName}"`)
+      );
+
+    this.setState({ appIsInstalled: true });
+
+    return {
+      targetState: {
+        EditorInterface: {
+          [contentType.sys.id]: {
+            controls: [
+              {
+                fieldId: 'imageMetadata',
+                settings: {
+                  imageFieldId: 'image'
+                }
+              }
+            ]
+          }
+        }
+      }
+    };
+  };
+
+  setContentTypeName = ({ target: { value } }) =>
+    this.setState({
+      contentTypeId: camelCase(value),
+      contentTypeName: value
+    });
+
+  render() {
+    const { appIsInstalled, allContentTypesIds, contentTypeId, contentTypeName } = this.state;
+
+    return (
+      <>
+        <div className={styles.background} />
+        <div className={styles.body}>
+          <div>
+            <Typography>
+              <Heading className={styles.heading}>About Image Wrapper</Heading>
+              <Paragraph>
+                This app assists you in managing image uploads that need to have metadata associated
+                with them (e.g. a focal point for better cropping, tags, alt text).
+              </Paragraph>
+              <Divider />
+              {appIsInstalled && <ConfigurationContent />}
+              {!appIsInstalled && (
+                <InstallationContent
+                  allContentTypesIds={allContentTypesIds}
+                  contentTypeId={contentTypeId}
+                  contentTypeName={contentTypeName}
+                  setContentTypeName={this.setContentTypeName}
+                />
+              )}
+            </Typography>
+          </div>
+        </div>
+        <div className={styles.logo}>
+          <img src="https://image.flaticon.com/icons/svg/189/189089.svg" alt="logo" />
```

Comment:
1. We should check that we can actually use this image.

2. We should include this file right in the package so we don't have it disappear on us in the future.

Reply:
This is a placeholder :) Malin is making a real logo it'll be removed